index.html

```diff
@@ -759,6 +742,21 @@ <h2>General</h2>
           </blockquote>
         </section>
 
+        <section id='ldpcvget'>
+          <h2>HTTP GET</h2>
+          <p>
+            An <a>LDPCv</a> MUST respond to <code>GET Accept: application/link-format</code> as indicated in
+            [[!RFC7089]] <a href='https://tools.ietf.org/html/rfc7089#section-5'>section 5</a> and specified in
+            [[!RFC6690]] <a href='https://tools.ietf.org/html/rfc6690#section-7.3'>section 7.3</a>.
+          </p>
+          <p>
+            An implementation MUST include the <code>Allow</code> header as outlined in
```

Comment:
This is only a style preference, but these three sentences would read better for me if they were each in their own '\<p\>' block.